.github/workflows/build.yml

```diff
@@ -131,6 +138,13 @@ jobs:
         run: |
           cmake --build build --target install
           cmake --build build --target dmg
+      - name: Display help
+        run: >
+          build/LMMS.app/Contents/MacOS/lmms --help | grep "Usage: lmms"
+      - name: Display version
+        run: >
+          build/LMMS.app/Contents/MacOS/lmms --version |
+          grep "Copyright (c) .* LMMS Developers"
```

Comment:
```suggestion
        run: build/LMMS.app/Contents/MacOS/lmms --version | grep "Copyright (c) .* LMMS Developers"
```

Reply:
This one also would get > 80, so yamllint would hate me for that change.

Reply:
Sorry, I just see we changed yamllint to allow 120. So will consider committing those proposals, too.